### PR TITLE
go modules, go 1.16, upgrade sdk & remove old src structure

### PR DESCRIPTION
**Pull request by ivancorrales**

### CODE REVIEW

File: tests/rabbitmq_test.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	envVars := []string{
+		fmt.Sprintf("HOSTNAME=%s", hostname),
+		"USERNAME=guest",
+		"PASSWORD=guest",
+	}
+	response, _, err := executeDockerCompose(containerName, envVars)
+	assert.Nil(t, err)
+	assert.NotEmpty(t, response)
+
+	assert.Equal(t, "com.newrelic.rabbitmq", gjson.Get(response, "name").String())
```
Comment (gsanchezgavier) [inline tests/rabbitmq_test.go:37]: why not using json schema helper?

File: cmd/nri-rabbitmq/main.go [hunk 2]
```diff
@@ -201,3 +205,43 @@ func exitOnError(err error) {
 		os.Exit(-1)
 	}
 }
+
+func alivenessTest(rabbitmqIntegration *integration.Integration, vhostTests []*data.VhostTest, clusterName string) {
```
Comment (gsanchezgavier) [inline cmd/nri-rabbitmq/main.go:209]: i don't see where this new functions are used in the PR, could you give some context 😄

Comment (ivancorrales) [reply cmd/nri-rabbitmq/main.go:209]: Sure, https://github.com/newrelic/nri-rabbitmq/blob/feature/update-sdk/cmd/nri-rabbitmq/main.go#L75-L76 Actually I haven't created these functions ... maybe I moved the from other file...

File: .github/workflows/prerelease.yml [hunk 3]
```diff
@@ -9,26 +9,20 @@ on:
...
-  GO_VERSION: '1.9.7'
+  GO_VERSION: '1.16'
   INTEGRATION: "rabbitmq"
   ORIGINAL_REPO_NAME: 'newrelic/nri-rabbitmq'
   REPO_FULL_NAME: ${{ github.event.repository.full_name }}
   TAG: ${{ github.event.release.tag_name }}
 
 jobs:
-
-  validate:
```
[the remark this answers is not in the document]
Comment (ivancorrales) [reply .github/workflows/prerelease.yml:20]: Fixed

**Review by gsanchezgavier — APPROVED** [review]
looks good!

**Pull request merged**